sysutils/ondir/Portfile

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem       1.0
+
+name             ondir
+version          0.2.3
+revision         1
+
+categories       sysutils
+platforms        darwin
+license          GLP-2
+
+maintainers      nomaintainer
+
+description      Automatic script execution on cd
+long_description OnDir is a program that automatically executes scripts as you traverse directories at a terminal.
+
+homepage         https://swapoff.org/ondir.html
+master_sites     https://swapoff.org/files/ondir/
+distfiles        ${name}-${version}.tar.gz
```

Comment:
Please do not set `distfiles` directly. Set the `distname` instead. However, this looks like the default. So this line should be deleted.

Reply:
removed the line! thanks again for your help! 
\--